README.md

```diff
@@ -133,7 +135,7 @@ Gets the item identified by `id`. Item `id`s are returned by [POST /run-process]
 
 ### GET /item/:id/metadata/:metadataKey
 
-Gets the metadata file matching the `metadataKey` for the the item identified by `id`. Item `id`s are returned by [POST /run-process](#post-/run-process). The file will be returned with a `Content-Type` of `application/octet-stream`. The original `filename` is returned in the `Content-Disposition` header.
+Gets the metadata value matching the `metadataKey` for the the item identified by `id`. Item `id`s are returned by [POST /run-process](#post-/run-process). If the value is a string, it will be returned with a `Content-Type` of `application/json`. If the value is a file, it will be returned with a `Content-Type` of `application/octet-stream`. The original `filename` is returned in the `Content-Disposition` header.
```

Comment:
Content-type shouldn't be `application/json` here I don't think. Probably better as `text/plain`

Reply:
changed to set `content/type: text/plain` + tests